clijs/test/commands/smart-account/deploy.test.ts

```diff
@@ -26,12 +28,37 @@ describe("smart-account:deploy", () => {
         "smart-account",
         "send-transaction",
         "-a",
-        "../nil/contracts/compiled/tests/Test.abi",
+        "../nil/contracts/compiled/tests/Counter.abi",
         contractAddress,
-        "setValue",
+        "add",
         "10",
       ])
     ).result as Hex;
     expect(txHash).toBeTruthy();
+
+    const resultValue = (
+      await runCommand([
+        "smart-account",
+        "call-readonly",
+        "-a",
+        "../nil/contracts/compiled/tests/Counter.abi",
+        contractAddress,
+        "value",
+      ])
+    ).result as string;
+    expect(resultValue).toEqual(10);
+
+    const estimation = (
+      await runCommand([
+        "smart-account",
+        "estimate-fee",
+        "-a",
+        "../nil/contracts/compiled/tests/Counter.abi",
+        contractAddress,
+        "add",
+        "20",
+      ])
+    ).result as Hex;
+    expect(estimation).greaterThan(0);
```

Comment:
I suppose it should be:
```suggestion
    expect(BigInt(estimation().greaterThan(0);
```